### PR TITLE
Fix inability to create organizer from Typeahead

### PR DESCRIPTION
**Pull request by Anahkiasen**
### Fixed

- Fix inability to create organizer from Typeahead

---

Also fixed the two warnings about the missing `id` prop and the `labelKey` which must be a string when using `allowNew` (didn't see this cause any issue when picking existing organizations).

Ticket: https://jira.uitdatabank.be/browse/III-5332

### CODE REVIEW

File: src/pages/OrganizerAddModal.tsx [hunk 1]
```diff
@@ -325,6 +325,7 @@ const OrganizerAddModal = ({
           <Title size={2}>Contact</Title>
           <ContactInfoStep
             isOrganizer={true}
+            onValidationChange={() => ({})}
```
Comment (brampauwelyn) [inline src/pages/OrganizerAddModal.tsx:328]: Why is this necessary?
If it's optional it should be ok right?

Comment (simon-debruijn) [reply src/pages/OrganizerAddModal.tsx:328]: Also `() => ({})` doesn't match the type specified `(status: ValidationStatus) => void;`

SInce you're returning an object instead of void

Comment (Anahkiasen) [reply src/pages/OrganizerAddModal.tsx:328]: I worked backward, since it was crashing because the prop was not provided I made it not optional, then provided it in a way that wouldn't crash, but it's probably better to amend the code to make it support the prop not being passed

**Review by simon-debruijn — APPROVED** [review]
